exercises/simple-linked-list/test/simple_linked_list_tests.erl

```diff
@@ -0,0 +1,91 @@
+-module(simple_linked_list_tests).
+
+-include_lib("erl_exercism/include/exercism.hrl").
+
+-include_lib("eunit/include/eunit.hrl").
+
+'1_value_reflects_constructor_arg_test'() ->
+    L = simple_linked_list:empty(),
+    L2 = simple_linked_list:cons(2, L),
+    ?assertMatch(2, (simple_linked_list:head(L2))).
+
+'2_new_lists_should_have_length_0_test'() ->
+    L = simple_linked_list:empty(),
+    ?assertMatch(0, (simple_linked_list:count(L))).
+
+'3_adding_a_element_increments_length_test'() ->
+    L = simple_linked_list:empty(),
+    L2 = simple_linked_list:cons(1, L),
+    ?assertMatch(1, (simple_linked_list:count(L2))).
```

Comment:
Perhaps as second combine this and current number 1, `assert`ing on the length and the head separetely?